README.md

```diff
@@ -0,0 +1,5 @@
+# PlatformDevSlides
+Slides to the platform dev course
+
+to clone this repo use:
+    `git clone https://github.com/NuclearCookie/PlatformDevSlides.git --recursive`
```

Comment:
You should probably add `--origin upstream`, such that your fork (for contributions) can be the remote `origin`.